<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SiirryDlg</class>
 <widget class="QDialog" name="SiirryDlg">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Siirry tositteeseen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pic/pic.qrc">
    <normaloff>:/pic/etsi.png</normaloff>:/pic/etsi.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="tyyppiLabel">
       <property name="text">
        <string>Tositelaji</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Numero</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Tilikausi</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="tyyppiCombo"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nroEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="kausiCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="siirryNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Siirry</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/etsi.png</normaloff>:/pic/etsi.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Peru</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/sulje.png</normaloff>:/pic/sulje.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>siirryNappi</sender>
   <signal>clicked()</signal>
   <receiver>SiirryDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SiirryDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
